repository/q.json

```diff
@@ -239,6 +239,19 @@
 				}
 			]
 		},
+		{
+			"name": "Quickfix List",
+			"details": "https://github.com/cibinmathew/sublime_quickfix_list",
+			"author": "Cibin Mathew",
+			"labels": ["navigation"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"platforms": ["osx", "linux"],
+					"branch": "master"
```

Comment:
Oh, wait, you need to remove the branch here and set it to `"tags": true`. We don't support branch based releases anymore, and you are properly tagging releases already.

Reply:
That's maybe also why the bot is failing.

Reply:
@braver I have made the mentioned changes